docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -77,18 +72,21 @@ To answer these questions, you will need to process the data using Sycamore’s
 
 `docker compose run sycamore_crawler_http_sort_all`
 
-Note: If you just want to ingest a few PDFs instead of the whole dataset, run:
+Note: If you just want to ingest one PDF instead of the whole dataset (to save time), run:
 
 `docker compose run sycamore_crawler_http_sort_one`
 
 Sycamore will automatically start processing the new data. The processing job is complete and the data is loaded into the index once you see log messages similar to:
 
 `No changes at [datetime] sleeping`
 
-3. Go to the demo query UI at localhost:3000. You can interact with the demo UI while data is being added to the index, but the data won't all be available until the job is done.
+3. Use the demo UI for conversational search. Using your internet browser, visit http://localhost:3000. You can interact with the demo UI while data is being added to the index, but the data won't all be available until the job is done.
 
+* Create a new conversation. Enter the name for your conversation in the text box in the left "Conversations" panel, and hit enter or click the "add convo" icon on the right of the text box.
```

Comment:
This step isn't needed anymore, but we can leave it in.

Reply:
What step? Creating a new convo?